apis/nucleus/src/components/listbox/components/__tests__/list-box-row-column.test.jsx

```diff
@@ -827,6 +827,23 @@ describe('<ListBoxRowColumn />', () => {
       await testRenderer.unmount();
     });
 
+    test('should get correct text alignment', async () => {
+      const index = 0;
+      const style = {};
+      const data = {
+        keyboard,
+        textAlign: { align: 'right' },
+      };
+      const testRenderer = await render(
+        <ThemeProvider theme={theme}>
+          <ListBoxRowColumn index={index} style={style} data={data} column={rowCol === 'column'} />
+        </ThemeProvider>
+      );
+      const testInstance = testRenderer.root;
+      const type = testInstance.findByType(Grid);
+      expect(type.props.children[1].props.style.justifyContent).toEqual('right');
+    });
+
```

Comment:
If possible, it would be great to also add tests for the other cases: align: 'left' and RTL.